app/chapters/[slug]/layout.tsx

```diff
@@ -1,13 +1,37 @@
-import Navbar from 'components/Navbar'
+'use client'
+
+import Navbar, { NavbarMobile } from 'components/Navbar'
 import chapters from 'content/chapters'
+import { useEffect, useState } from 'react'
+
+function getWindowWidth() {
+  const { innerWidth: width } = window
+  return width
+}
 
-export default async function Layout({ children, params }) {
+export default function Layout({ children, params }) {
   const isValidChapter = chapters[params.slug]
 
+  const desktopSize = 800
```

Comment:
Instead of `800` we should use `767` as the breaking point for mobile UI, we are already utilizing 767 for several components and pages

Reply:
Probably ideal if this number is whatever Tailwind is configured to for responsive sizes.

Reply:
Yes, we should use the default Tailwind screen size breakpoints - https://tailwindcss.com/docs/screens#fixed-range-breakpoints

Reply:
Sure @vinayaksh42. Let me fix this up!

Reply:
Addressed in b5210a1c3c0612450de52ddea543f2984248d6bd